jekyll/_cci2/insights-partnerships.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: classic-docs
+title: "Insights Partnerships"
+short-title: "Insights Partnerships"
+version:
+- Cloud
+---
+
+## Overview
+
+{:toc}
+
+This document describes how you can connect Insights data with third party
+providers.
+
+### Sumo Logic integration
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 9. You can fix by inserting `{: #sumo-logic-integration }` to the next line